docker-compose.yml

```diff
@@ -8,10 +8,10 @@ services:
     ports:
       - "8081:8080"
     healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:8080/health/ping"]
+      test: ["CMD", "curl", "-f", "http://localhost:8080/ping"]
```

Comment:
ping is usually under health and supplied by Spring Boot, might be worth checking that out

Reply:
Yes it is. Delete the whole ping thing.